patches/third_party-blink-renderer-core-dom-events-event_target.cc.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/third_party/blink/renderer/core/dom/events/event_target.cc b/third_party/blink/renderer/core/dom/events/event_target.cc
+index 273da6dbe36669bd1a8daf2303807b34e4962b12..adf0363171c71e71e08d8cf666b9cc0d4fdd4384 100644
+--- a/third_party/blink/renderer/core/dom/events/event_target.cc
++++ b/third_party/blink/renderer/core/dom/events/event_target.cc
+@@ -460,6 +460,7 @@ bool EventTarget::AddEventListenerInternal(
+   bool added = EnsureEventTargetData().event_listener_map.Add(
+       event_type, listener, options, &registered_listener);
+   if (added) {
++    BRAVE_EVENT_TARGET_ADD_EVENT_LISTENER_INTERNAL
```

Comment:
```#define AddedEventListener(EVENT_TYPE, LISTENER) AddedEventListener(EVENT_TYPE, LISTENER); ...``` ?

Reply:
it won't work. `AddedEventListener` is implemented right here as `void EventTarget::AddedEventListener`.

Reply:
that's odd, I swear these were implemented in a base class. Maybe it changed in a cr update or maybe I just missed it somehow.